### PR TITLE
Add haikunator and phoenix_html_sanitizer

### PR DESCRIPTION
**Pull request by rrrene**
Adds the two packages, therefore fixing #847 and #849 :grinning:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -390,6 +390,7 @@ A curated list of amazingly awesome Elixir libraries, resources, and shiny thing
 * [phoenix_ecto](https://github.com/phoenixframework/phoenix_ecto) - Phoenix and Ecto integration.
 * [phoenix_generator](https://github.com/etufe/phoenix_generator) - Generate boilerplate for Phoenix applications.
 * [phoenix_html](https://github.com/phoenixframework/phoenix_html) - Phoenix.HTML functions for working with HTML strings and templates.
+* [phoenix_html_sanitizer](https://github.com/elixirstatus/phoenix_html_sanitizer) - HTML Sanitizer integration for Phoenix
```
Comment (doomspork) [inline README.md]: You're missing a period here.

**Comment by rrrene** [comment]
@doomspork Thanks for the hint!

**Comment by rrrene** [comment]
@h4cc This PR still fails, but from [the output](https://travis-ci.org/h4cc/awesome-elixir/builds/75682037#L4988) I think that it is not really due to my changes ...

**Comment by h4cc** [comment]
Yep, looks good.

**Comment by h4cc** [comment]
Thanks!

**Pull request merged**